pom.xml

```diff
@@ -32,118 +32,9 @@
     </properties>
 
     <modules>
-        <module>digiwf-engine</module>
         <module>digiwf-connector</module>
-        <module>digiwf-schema-registry</module>
-        <module>digiwf-task</module>
-        <module>digiwf-libs</module>
-        <module>digiwf-process</module>
-        <module>digiwf-gateway</module>
-        <module>digiwf-integrations</module>
-        <module>digiwf-coverage</module>
     </modules>
```

Comment:
Genau dieser Teil muss komplett raus. Das ist die Aggregation und kann eigenständig in einer POM gelöst sein.

Dann hat man zwei weitere POMs, die nur als Parent dienen mit den unterschiedlichen groupIDs, aber alle Module weiterhin in der Aggregation-POM zusammen gebaut über einen Befehl